src/style.scss

```diff
@@ -1,10 +1,12 @@
 /**
- * The following styles get applied both on the front of your site
- * and in the editor.
- *
- * Replace them with your own styles or remove the file completely.
+ * Block front-end and editor styles.
  */
 
 .wp-block-jartes-wikiblock {
 	padding: 2px;
 }
+
+.wp-block-jartes-wikiblock iframe {
+	max-width: 100%;
+	width: 100%;
```

Comment:
For me on FF, this means the iframe doesn't respect whatever I set in the width attribute. If you want to keep a responsive version, you could add that as another option, which sets a special class on the iframe.

Reply:
Correct, and I will remove this as this was intended for the built-in block width option that I'll add later :)